javalib/src/main/scala/java/util/NavigableView.scala

```diff
@@ -121,22 +124,23 @@ private[util] class NavigableView[E](original: NavigableSet[E],
 
   def last(): E = {
     val iter = iterator()
-    if (iter.hasNext) iter.asScala.toTraversable.last
+    if (iter.hasNext) scala.collection.Iterable.from(iter.asScala).last
     else null.asInstanceOf[E]
   }
 
   def subSet(fromElement: E, fromInclusive: Boolean, toElement: E,
       toInclusive: Boolean): NavigableSet[E] = {
     val innerNow = inner()
+    implicit val boxOrdering: Ordering[Box[E]] = innerNow.ordering
```

Comment:
Because the `mutable.SortedSet.from(…)` below (line 143) requires the implicit ordering instance. This is going to disappear with your suggestion of using `….clone() -= …`.